README.md

```diff
@@ -14,87 +14,44 @@ This project offers environment for all the labs in **Red Hat RHCSA/RHCE 7 Cert
 - [Vagrant](https://www.vagrantup.com): tool for building complete development environments. With an easy-to-use workflow and focus on automation
 
 Don't worry you don't have to be expert on any of these tools.
-### 1. Install virtualbox :
 
-**1.1 Install virtualbox on Ubuntu:**
+## install Vagrant
+follow the individual instructions for your platform:
+- RHEL/Centos 7 with [VirtualBox](rhel-centos7-vbx.md) or [KVM](rhel-centos7-kvm.md)
+- [Fedora](fedora-vbx.md)
+- macOS TODO
+- [Ubuntu](ubuntu.md)
+- Windows TODO
 
-```shell
-$ wget -q -O - http://download.virtualbox.org/virtualbox/debian/oracle_vbox_2016.asc | sudo apt-key add -
-
-$ sudo sh -c 'echo "deb http://download.virtualbox.org/virtualbox/debian `lsb_release -sc` non-free contrib" > /etc/apt/sources.list.d/virtualbox.org.list'
-
-$ sudo apt-get update
-
-$ sudo apt-get install dkms
-
-$ sudo apt-get install virtualbox-5.1
-```
-**1.2 Install virtualbox on Oracle Linux/RHEL/CentOS:**
-```shell
-$ wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | rpm --import -
-
-$ sudo wget http://download.virtualbox.org/virtualbox/rpm/el/virtualbox.repo -O /etc/yum.repos.d/virtualbox.repo
-
-$ sudo yum install VirtualBox-5.1
-```
 
-**1.3 Install virtualbox on Fedora:**
-```shell
-$ wget -q https://www.virtualbox.org/download/oracle_vbox.asc -O- | rpm --import -
-
-$ sudo wget http://download.virtualbox.org/virtualbox/rpm/fedora/virtualbox.repo -O /etc/yum.repos.d/virtualbox.repo
-
-$ sudo yum install VirtualBox-5.1
-```
-### 2. Install vagrant :
-
-**2.1 Install vagrant on Debian-based Linux Ubuntu/Mint (64-bit):**
-```shell
-$ wget https://releases.hashicorp.com/vagrant/1.8.7/vagrant_1.8.7_x86_64.deb
-
-$ sudo dpkg -i vagrant_1.8.6_i686.deb
-```
-
-**2.2 Install vagrant on Debian-based distributions Ubuntu/Mint (32-bit):**
-```shell
-$ wget https://releases.hashicorp.com/vagrant/1.8.7/vagrant_1.8.7_i686.deb
-
-$ sudo dpkg -i vagrant_1.8.6_i686.deb
-```
-
-**2.3 Install vagrant on RedHat-based distributions (64-bit):**
-```shell
-$ sudo yum -y install https://releases.hashicorp.com/vagrant/1.8.7/vagrant_1.8.7_x86_64.rpm
-```
+## How to start servers?
+Navigate to project path where Vagranfile exists.
 
-**2.4 Install vagrant on RedHat-based distributions (32-bit):**
+- setup all servers
 ```shell
-$ sudo yum -y install https://releases.hashicorp.com/vagrant/1.8.7/vagrant_1.8.7_i686.rpm
+cd ~/RHCSA-RHCE-Lab-Environment
+vagrant up                  # create all machines, may take 10-60 minutes
+ssh-keygen -R 192.168.4.200; ssh-copy-id root@192.168.4.200
+ssh-keygen -R 192.168.4.210; ssh-copy-id root@192.168.4.210
+ssh-keygen -R 192.168.4.220; ssh-copy-id root@192.168.4.220
+vagrant halt                # shutdown all machines
+vagrant status              # all 4 have state poweroff
+vagrant snapshot save begin # create a VM snapshot
```

Comment:
Thank you, in case of errors you can give it a clean start. Should also work on platforms other than Virtualbox.

Sander sometimes assumes changes made in previous chapters, sometimes he doesn't. IMHO a _candidate_ RHCSA should be able to make a small script or flip back a few pages, to get _on the same page_ were Sander assumes you are.